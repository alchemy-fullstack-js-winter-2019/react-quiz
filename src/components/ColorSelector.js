import React from 'react';
import PropTypes from 'prop-types';



function ColorSelector({ handleChange }) {

    return (
        <>
            <label>FONT COLOR 
                <input type='color' name="FontColor" onChange={handleChange}/>
            </label>
            <label>BACKGROUND COLOR 
                <input type='color' name="BackgroundColor" onChange={handleChange}/>
            </label>
        </>
    );

}

ColorSelector.propTypes = {
    handleChange: PropTypes.func.isRequired
};


export default ColorSelector;
