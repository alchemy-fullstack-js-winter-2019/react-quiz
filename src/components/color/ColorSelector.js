import React, { Fragment } from 'react';

function ColorSelector() {
  return (
    <Fragment>
      <h2>I am colors</h2>
    </Fragment>
  );
}

export default ColorSelector;
