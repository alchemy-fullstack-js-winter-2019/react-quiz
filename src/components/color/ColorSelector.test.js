import React from 'react';
import renderer from 'react-test-renderer';
import ColorSelector from './ColorSelector';

describe('ColorSelector', () => {
  it('matches a snapshot', () => {
    const tree = renderer.create(
      <ColorSelector />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
