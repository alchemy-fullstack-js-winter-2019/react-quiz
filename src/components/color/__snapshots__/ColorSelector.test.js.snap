// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorSelector matches a snapshot 1`] = `
<form>
  <label>
    <span>
      Font Color:
    </span>
    <input
      name="fontColor"
      type="color"
    />
  </label>
  <label>
    <span>
      Background Color:
    </span>
    <input
      name="backgroundColor"
      type="color"
    />
  </label>
</form>
`;
